TASK/TaskFollowNavMeshToCoord.md

```diff
@@ -40,9 +40,9 @@ To identify when this has happened, you can use GET_NAVMESH_ROUTE_RESULT. This w
 * **x**: X Position Coordinate.
 * **y**: Y Position Coordinate.
 * **z**: Z Position Coordinate
-* **moveBlendRatio**: 
+* **moveBlendRatio**: Ratio for ped movement(?)
 * **time**: INT value in milliseconds.
-* **radius**: 
+* **radius**: Radius of how far the ped moves(?)
```

Comment:
`?` why?

Reply:
not entirely sure as to if this is correct or that its a good description for them